mlir/lib/IR/AffineExpr.cpp

```diff
@@ -117,6 +117,122 @@ bool AffineExpr::isSymbolicOrConstant() const {
   llvm_unreachable("Unknown AffineExpr");
 }
 
+bool AffineExpr::isSymbolic() const {
+  switch (getKind()) {
+  case AffineExprKind::SymbolId:
+    return true;
+  case AffineExprKind::Constant:
+  case AffineExprKind::DimId:
+  case AffineExprKind::Add:
+  case AffineExprKind::Mul:
+  case AffineExprKind::FloorDiv:
+  case AffineExprKind::CeilDiv:
+  case AffineExprKind::Mod:
+    return false;
```

Comment:
Right now the `switch` and the `case`s lie in the same indentation level. I believe the `cases` should have another level of indentation.

Reply:
Right, but in the same file, a similar pattern is followed and it is in clang-format also. So for uniformity, I didn't change the pattern.

Reply:
Yes, as per the LLVM style, they don't indent case's - it's just to save horizontal space for this special situation.

Reply:
Oh okay, I see.